net.ptilouk.superfluous.yml

```diff
@@ -0,0 +1,117 @@
+app-id: net.ptilouk.superfluous
```

Comment:
can you add a link to the GitHub in the website

Reply:
A link to what? My Github page or the Github repository of the game?

Reply:
Alright, I added a link to the Github repo on the FLOSS page of the game: https://studios.ptilouk.net/superfluous-returnz/floss/